package.json

```diff
@@ -10,30 +10,52 @@
   "license": "MIT",
   "author": "Sanity.io <hello@sanity.io>",
   "sideEffects": false,
+  "type": "commonjs",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/1.esm.d.ts",
-        "default": "./dist/1.esm.mjs"
+      "types": "./dist/index.d.ts",
+      "source": "./src/1.ts",
+      "require": "./dist/index.js",
+      "node": {
+        "import": "./dist/index.cjs.mjs",
```

Comment:
This is a generated file by `@sanity/pkg-utils` that looks like this:
```js
import cjs from './index.js';

export const evaluate = cjs.evaluate;
export const parse = cjs.parse;

```

This Node.js specific problem this technique solves is known as the [Dual Package Hazard](https://nodejs.org/api/packages.html#dual-package-hazard) and is known to cause `instanceof` checks and similar to break if both the ESM and CJS version of a package is loaded up and used interchangeably in an app. Usually due to long chains of dependencies of dependencies that go in and out of the CJS and ESM runtime.